src/dolphin/install/installation.ts

```diff
@@ -141,7 +141,7 @@ export class DolphinInstallation {
         dolphinDownloadInfo = await fetchLatestVersion(type);
       } catch (err) {
         log.error(`Failed to fetch latest Dolphin version: ${err}`);
-        return;
+        throw new Error(`Failed to fetch latest Dolphin version`);
```

Comment:
I'm confused. Why even bother surrounding it in try-catch if we're just gonna throw. You could probably just do away with the try-catch block.